x/uibc/msg_test.go

```diff
@@ -0,0 +1,96 @@
+package uibc
+
+import (
+	"testing"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"gotest.tools/assert"
+)
+
+func TestMsgGovUpdateQuota(t *testing.T) {
+	tests := []struct {
+		name          string
+		title         string
+		authority     string
+		description   string
+		total         sdk.Dec
+		perDenom      sdk.Dec
+		quotaDuration time.Duration
+		errExpected   bool
+	}{
+		{
+			name:          "valid msg",
+			title:         "update quota",
+			authority:     authtypes.NewModuleAddress("gov").String(),
+			description:   "desc",
+			total:         sdk.MustNewDecFromStr("1000"),
+			perDenom:      sdk.MustNewDecFromStr("1000"),
+			quotaDuration: 100,
+			errExpected:   false,
+		},
+		{
+			name:          "invalid authority address in msg",
+			title:         "update quota",
+			authority:     authtypes.NewModuleAddress("govv").String(),
+			description:   "desc",
+			total:         sdk.MustNewDecFromStr("1000"),
+			perDenom:      sdk.MustNewDecFromStr("1000"),
+			quotaDuration: 100,
+			errExpected:   true,
+		},
+	}
```

Comment:
if we want to add more test cases, then let's use a helper function. See for example: https://github.com/umee-network/umee/blob/main/x/leverage/types/token_test.go#L104